### PR TITLE
fix:changed the static final variable into enum

### LINKED ISSUE #1034: changing the static final variables to enum in wave generator activity
**Actual Behaviour**

In wave generator activity there are many static final variables.

**Expected Behaviour**

These variables can be replaced by java enum.

**Steps to reproduce it**

Change all the variable in Wave Generator Activity and Wave Generator Common

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Avjeet**
Fixes #1034 

**Changes**: Changed all the static final variable in both WaveGenerator and WaveGeneratorCommon to using java enum constants.

Note:- This PR requires the PR #1010  to be merged first.

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [ ] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: [app-debug.zip](https://github.com/fossasia/pslab-android/files/2102814/app-debug.zip)

### CODE REVIEW

File: app/src/main/java/org/fossasia/pslab/activity/WaveGeneratorActivity.java [hunk 1]
```diff
@@ -67,18 +105,31 @@
...
-    public static final String FREQUENCY = "frequency";
-    public static final String PHASE = "phase";
-    public static final String DUTY = "duty";
-    private static String btn_Active = "";
-    private static String prop_active = "";
+
+    private WaveConst waveBtnActive, pwmBtnActive, prop_active;
+    private static boolean waveMonSelected;
+    private TextView activePropTv = null;
+    private TextView activePwmPinTv = null;
```
Comment (Vikum94) [inline app/src/main/java/org/fossasia/pslab/activity/WaveGeneratorActivity.java]: Where is this view bidden to? Is it in working progress?

Comment (Avjeet) [reply app/src/main/java/org/fossasia/pslab/activity/WaveGeneratorActivity.java]: That just textview holder which assumes the textview already on the screen so as to changes their values.

Comment (Avjeet) [reply app/src/main/java/org/fossasia/pslab/activity/WaveGeneratorActivity.java]: Yes it is in working state.

**Comment by Avjeet** [comment]
@Vikum94 plz approve if there is no other change.
@abhinavraj23 @harsh-2711 @CloudyPadmal plz review it

**Pull request merged**